packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1020,6 +990,7 @@ task(TASK_NODE)
     "Completely disable all of ChugSplash's activity."
   )
   .addFlag('hide', "Hide all of ChugSplash's output")
+  .addFlag('confirm', 'Automatically confirm contract upgrades')
```

Comment:
Since the `node` task always runs on the Hardhat network, we can remove the 'confirm' flag and let `confirm` be `true` in the call to `deployAllChugSplashConfigs`.